deploy-all

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env ruby
+
+"""
```

Comment:
This looks Python-y :)  Can it be a comment instead? rubocop complains about the triple quotes.

Reply:
Fixed in 6be96c1